vaadin-dialog-flow-parent/vaadin-dialog-flow-integration-tests/src/main/java/com/vaadin/flow/component/dialog/tests/DialogTestPage.java

```diff
@@ -197,19 +203,33 @@ private void createResizableDraggableDialog() {
         message.setId("dialog-resizable-draggable-message");
 
         dialog.addResizeListener(e -> message.setText(String.format(
```

Comment:
Looks like there are no tests that verify that `top`, `left`, `width`, `height` are synced back to the dialog on `resize` or `dragged`. Would be good to add those.

Could also add a plain unit test for that IMO.

Reply:
Added a couple of unit tests to check each event's default synching behavior.